docs/wfc3tools/wf3cte.rst

```diff
@@ -57,15 +99,11 @@ Parameters
 
             * a single filename (``iaa012wdq_raw.fits``)
             * a Python list of filenames
-            * a partial filename with wildcards (``\*raw.fits``)
-            * filename of an ASN table (``\*asn.fits``)
+            * a partial filename with wildcards (``*raw.fits``)
             * an at-file (``@input``)
 
-    output : str, default=None
```

Comment:
is output no longer an arg?

Reply:
The "output" parameter was removed as it has always actually
been essentially an ignored parameter.  The C API for this 
component of the pipeline improperly accommodated an "output" 
filename string.  However, if one digs deeper into the code, the 
final output name is actually a construction based upon the raw 
input filename.  The reason the "output" string was passed as a 
parameter to the lower-level routines is that the output
variable was allocated space (with its null termination) in
the higher-level routine. Since the Python WFC3TOOLS wrapper
is a thin interface to the C component, the wrapper also 
contains this improper parameter.

Reply:
Addressed in 6101480.